homeassistant/components/yamaha/media_player.py

```diff
@@ -195,7 +195,7 @@ def __init__(self, name, receiver, source_ignore, source_names, zone_names):
         self.receiver = receiver
         self._muted = False
         self._volume = 0
```

Comment:
Other properties can be adjusted too, let's make sure we keep the inner workings of integrations consistent (if possible).